README.md

```diff
@@ -241,7 +241,10 @@ First make sure your host has
   It also should use `/var/run/docker.sock` as socket to interact with the daemon (or you
   will have to override in `<absolute-project-path>/fabric-private-chaincode/config.override.mk` the default definition in make of `DOCKER_DAEMON_SOCKET`)
 * GNU make
-
+Then clone this git repository to your local machine taking special care to specify -recursive as follows:
+```bash
+git clone --recursive https://github.com/hyperledger-labs/fabric-private-chaincode.git $GOPATH/src/github.com/hyperledger-labs/fabric-private-chaincode
+```
```

Comment:
as discussed in our community call. We should move the section how to clone FPC which we already have to the beginning of the Getting started section instead of repeating here again.

Reply:
Done!